source/vulkan/vulkan_impl_device.cpp

```diff
@@ -208,6 +208,8 @@ bool reshade::vulkan::device_impl::check_capability(api::device_caps capability)
 		return true;
 	case api::device_caps::shared_resource_nt_handle:
 		return !is_windows7();
+	case api::device_caps::resolve_depth_stencil:
+		return true;
```

Comment:
It could also be implemented using render passes, instead of just dynamic rendering, but can just go with dynamic rendering only for now. It is definetely necessary to check support for that though, as e.g. there is no NV driver on Windows 7 that supports it. Can do this pretty easily:
```c++
case api::device_caps::resolve_depth_stencil:
    return _dynamic_rendering_ext;
```
Technically also need to check if the `VK_KHR_depth_stencil_resolve` extension is enabled or Vulkan 1.2 is used. ReShade only forces Vulkan 1.1, so that doesn't guarantee that this is available. But I can add that in a follow up.